src/utils/index.ts

```diff
@@ -24,10 +24,10 @@ import { TokenPricesUSD } from "../state/application"
 import { getAddress } from "@ethersproject/address"
 
 export function isSynthAsset(chainId: ChainId, tokenAddress: string): boolean {
-  return (
-    chainId === ChainId.MAINNET &&
-    SYNTHETIX_TOKENS.includes(tokenAddress.toLowerCase())
-  )
+  if (chainId in SYNTHETIX_TOKENS) {
+    return SYNTHETIX_TOKENS[chainId]?.includes(tokenAddress) || false
```

Comment:
did you maybe want `return SYNTHETIX_TOKENS?.[chainId].includes(tokenAddress) || false` here?

Reply:
Good call out! TS was being picky about this with the old types I was using, but your solution will work